src/dmd/impcnvtab.d

```diff
@@ -14,17 +14,17 @@ module dmd.impcnvtab;
 
 import dmd.mtype;
 
-immutable ENUMTY[TMAX][TMAX] impcnvResult = impCnvTab.impcnvResultTab;
-immutable ENUMTY[TMAX][TMAX] impcnvType1 = impCnvTab.impcnvType1Tab;
-immutable ENUMTY[TMAX][TMAX] impcnvType2 = impCnvTab.impcnvType2Tab;
+immutable TY[TY.MAX][TY.MAX] impcnvResult = impCnvTab.impcnvResultTab;
+immutable TY[TY.MAX][TY.MAX] impcnvType1 = impCnvTab.impcnvType1Tab;
+immutable TY[TY.MAX][TY.MAX] impcnvType2 = impCnvTab.impcnvType2Tab;
 
 private:
 
 struct ImpCnvTab
 {
-    ENUMTY[TMAX][TMAX] impcnvResultTab;
-    ENUMTY[TMAX][TMAX] impcnvType1Tab;
-    ENUMTY[TMAX][TMAX] impcnvType2Tab;
+    TY[TY.MAX][TY.MAX] impcnvResultTab;
```

Comment:
`ENUMTY` was `int`, `TY` is now `ubyte`, meaning the size of `ImpCnvTab` is now changed. Not sure if that's a problem.

Reply:
I don't think it's a problem because
- the `impcntab` tables are generated at CTFE and `immutable`
- C++ already sees them as `ubyte` tables:

https://github.com/dlang/dmd/blob/b8642c4f7d62cc94d12fee36fbce1f7617692e3c/src/dmd/mtype.h#L129-L134

Reply:
As long as it works I'm happy. Otherwise `alias ENUMTY = int;` can be added.